lib/AutoEnforcer/BackingContainers.cpp

```diff
@@ -0,0 +1,104 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/Support/Error.h"
+
+#include "revng/AutoEnforcer/AutoEnforcerErrors.h"
+#include "revng/AutoEnforcer/BackingContainers.h"
+
+using namespace AutoEnforcer;
+using namespace llvm;
+using namespace std;
+
+BackingContainers
+BackingContainers::cloneFiltered(const BackingContainersStatus &Targets) {
+  BackingContainers Container;
+  for (const auto &pair : Containers) {
+    const auto &ContainerName = pair.first();
+    const auto &BackingContainer = *pair.second;
```

Comment:
C++20 structured bindings

Reply:
i can't, because first is not a member variable but a member function